src/engine/blkstore/blkstore.hpp

```diff
@@ -325,26 +325,31 @@ class BlkStore {
     BlkAllocStatus alloc_contiguous_blk(const uint32_t size, blk_alloc_hints& hints, BlkId* const out_blkid) {
         // Allocate a block from the device manager
         assert(size % m_pagesz == 0);
-        const blk_count_t nblks{static_cast< blk_count_t >(size / m_pagesz)};
+        const uint32_t nblks{static_cast< uint32_t >(size / m_pagesz)};
```

Comment:
This doesn't make sense to me. We change this from a 16bit value to a 32bit value (to avoid overflow?) and then later immediately cast it back to a 16bit parameter in the call to `alloc_contig...`; what's the point?